docs/applications/index.rst

```diff
@@ -45,28 +49,21 @@ To learn how to develop applications for Phalanx, see the :doc:`/developers/inde
 
 .. toctree::
    :maxdepth: 1
-   :caption: RSP+
+   :caption: rubin: Additional Rubin services
```

Comment:
I noticed that this uses a colon before the description and the others use parentheses. If we do want to keep the explanations we'd need to make this format consistent.